apps/testingaccessibility/src/hooks/use-nav-state.ts

```diff
@@ -1,14 +1,14 @@
 import {useCurrentAbility} from './use-current-ability'
+import {useUser} from './use-user'
 
 /**
  * TODO: consider removing this and using abilities?
  */
 export function useNavState() {
-  const {status, ability} = useCurrentAbility()
+  const {ability} = useCurrentAbility()
 
   return {
-    isSignedIn: status === 'authenticated',
-    isLoadingUser: status === 'loading',
```

Comment:
the loading aspect here was the only thing that wasn't an ability related shortcut, so I moved it out since passing it down seemed redundant